README.md

```diff
@@ -97,22 +118,17 @@ $ fh list releases nixos/nixpkgs
 ...
 ```
 
-### Listing organizations and flakes
+### Listing organizations, flakes, and versions
 
-`fh list orgs` and `fh list flakes` enumerates orgs and flakes on FlakeHub:
+[`fh list flakes`](#list-flakes), [`fh list orgs`](#list-flakes), and [`fh list versions`](#list-versions) enumerate [organizations][orgs], [flakes], and [flake versions][semver] on FlakeHub, respectively.
```

Comment:
This "respective" list seems wrong? It says `list flakes`, then `list orgs`, then `list versions`, but then basically says "enumerate orgs, flakes, versions"